packages/config/config/devices/0x0086/templates/aeotec_template.json

```
@@ -0,0 +1,1132 @@
+				"value": 1
+			},
+			{
+				"label": "Add Mode",
+				"value": 2
+			},
+			{
+				"label": "Remove Mode",
+				"value": 3
+			},
+			{
+				"label": "Association Mode",
+				"value": 4
+			},
+			{
+				"label": "Learn Mode",
+				"value": 5
+			}
+		]
+	},
+	"base_0-65535": {
+		"valueSize": 4,
+		"minValue": 0,
+		"maxValue": 65535,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"base_interval": {
+		"valueSize": 4,
+		"unit": "seconds",
+		"minValue": 1,
+		"maxValue": 2678400,
+		"defaultValue": 600,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"notification_type": {
+		"label": "Notification Type",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 11,
+		"defaultValue": 6,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Smoke alarm",
+				"value": 1
+			},
+			{
+				"label": "CO alarm",
+				"value": 2
+			},
+			{
+				"label": "CO2 alarm",
+				"value": 3
+			},
+			{
+				"label": "Heat alarm",
+				"value": 4
+			},
+			{
+				"label": "Water alarm",
+				"value": 5
+			},
+			{
+				"label": "Access control",
+				"value": 6
+			},
+			{
+				"label": "Home security",
+				"value": 7
+			},
+			{
+				"label": "Power management",
+				"value": 8
+			},
+			{
+				"label": "System notification",
+				"value": 9
+			},
+			{
+				"label": "Emergency alarm notification",
+				"value": 10
+			},
+			{
+				"label": "Clock notification",
+				"value": 11
+			}
+		]
+	},
+	"motion_report_type": {
+		"label": "Motion Sensor Triggered Command",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 2,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send Basic Set CC",
+				"value": 1
+			},
+			{
+				"label": "Send Sensor Binary Report CC",
+				"value": 2
+			}
+		]
+	},
+	"sensor_report_type": {
+		"label": "Motion Sensor Triggered Command",
+		"valueSize": 4,
+		"minValue": 16,
+		"maxValue": 256,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Send Sensor Binary Report CC",
+				"value": 16
+			},
+			{
+				"label": "Send Basic Set CC",
+				"value": 256
+			}
+		]
+	},
+	"trigger_off": {
+		"label": "Trigger Off Period",
+		"description": "Period after which to send Trigger Off command",
+		"valueSize": 2,
+		"unit": "seconds",
+		"minValue": 1,
+		"maxValue": 15300,
+		"defaultValue": 240,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"selective_reporting": {
+		"$import": "../../templates/master_template.json#base_enable_disable",
+		"label": "Selective Reporting",
+		"description": "Report only when the threshold defined below is exceeded"
```

```suggestion
		"description": "Report only when the defined threshold is exceeded"
```
"below" assumes knowledge about the UI we don't have.